examples/customresources/demos/complexmodules/client/client.go

```diff
@@ -0,0 +1,79 @@
+// Package main tests out all 2 custom models in the complexmodules.
+package main
+
+import (
+	"context"
+
+	"go.viam.com/rdk/examples/customresources/apis/gizmoapi"
+	"go.viam.com/rdk/examples/customresources/apis/summationapi"
+	"go.viam.com/rdk/logging"
+	"go.viam.com/rdk/robot/client"
+)
+
+func main() {
+	logger := logging.NewLogger("client")
+	robot, err := client.New(
+		context.Background(),
+		"localhost:8080",
+		logger,
+	)
+	if err != nil {
+		logger.Fatal(err)
+	}
+	//nolint:errcheck
+	defer robot.Close(context.Background())
```

Comment:
[nit] might as well fatally log an inability to `Close`.

Reply:
I'm generally against log.Fatal(). It prevents any defers from happening, including ones non-obvious because they live in deeply nested libraries we're using or similar. Yeah, probably fine here, with a single, standalone client app, but most of the time, it's equivalent to panic() in the "just blow it up" as an exit strategy kind of way.